contracts/src/TeleporterTokenDestination.sol

```diff
@@ -556,4 +570,77 @@ abstract contract TeleporterTokenDestination is
         }
         return SafeERC20TransferFrom.safeTransferFrom(IERC20(feeTokenAddress), feeAmount);
     }
+
+    function _validateSingleHopInput(
+        address destinationBridgeAddress,
+        uint256 secondaryFee,
+        address multiHopFallback
+    ) private view {
+        require(
+            destinationBridgeAddress == tokenSourceAddress,
+            "TeleporterTokenDestination: invalid destination bridge address"
+        );
+        require(secondaryFee == 0, "TeleporterTokenDestination: non-zero secondary fee");
+        require(
+            multiHopFallback == address(0),
+            "TeleporterTokenDestination: non-zero multi-hop fallback"
+        );
+    }
+
+    function _validateMultiHopInput(
+        address destinationBridgeAddress,
+        bytes32 destinationBlockchainID,
```

Comment:
Can we put `destinationBlockchainID` as the first param here before `destinationBridgeAddress` to keep the ordering consistent with all of the other places we specify each of those values?